### PR TITLE
Add the ElmFlutterView's width/height getter methods

### PR DESCRIPTION
**Pull request by JSUYA**
Add an API that returns the width and height size of evas_object.
and when resizing, compare the actual size, not the initial size.

### CODE REVIEW

**Comment by swift-kim** [comment]
Do you really need the new embedder APIs? How about getting the values from `evas_object_` directly?

**Comment by bbrto21** [comment]
@JSUYA If the TizenView consists of some container for wrapping image view and an image view, shouldn't they always have the same size?

**Comment by swift-kim** [comment]
@JSUYA That's is just an implementation detail. The user should only care about `evas_object_` (container) which is the only visible object from the outside.

File: embedding/cpp/elm_flutter_view.cc [hunk 1]
```diff
@@ -54,13 +56,32 @@ bool ElmFlutterView::RunEngine() {
...
+
+  int32_t view_width, view_height;
+  evas_object_geometry_get(evas_object_, nullptr, nullptr, &view_width,
+                           &view_height);
+  if (view_width != width || view_height != height) {
+    FlutterDesktopViewResize(view_, width, height);
   }
 }
 
+int32_t ElmFlutterView::width() {
```
Comment (swift-kim) [inline embedding/cpp/elm_flutter_view.cc:69]: This is not an accessor.
```suggestion
int32_t ElmFlutterView::GetWidth() {
```

Comment (JSUYA) [reply embedding/cpp/elm_flutter_view.cc]: Thanks. Done

**Comment by JSUYA** [comment]
> @JSUYA If the TizenView consists of some container for wrapping image view and an image view, shouldn't they always have the same size?

@bbrto21 Yes, image object and container are always the same size.

> @JSUYA That's is just an implementation detail. The user should only care about evas_object_ (container) which is the only visible object from the outside.

@swift-kim  OK, Thank you for your comment.

**Review by swift-kim — APPROVED** [review]

File: embedding/cpp/elm_flutter_view.cc [hunk 2]
```diff
@@ -54,13 +56,32 @@ bool ElmFlutterView::RunEngine() {
 }
 
 void ElmFlutterView::Resize(int32_t width, int32_t height) {
-  if (view_ && (width_ != width || height_ != height)) {
-    width_ = width;
-    height_ = height;
-    FlutterDesktopViewResize(view_, width_, height_);
+  assert(IsRunning());
+
+  int32_t view_width, view_height;
```
Comment (bbrto21) [inline embedding/cpp/elm_flutter_view.cc:61]: Could you please initialize local variable?

Comment (JSUYA) [reply embedding/cpp/elm_flutter_view.cc]: i added initialize

**Review by bbrto21 — APPROVED** [review]

**Comment by bbrto21** [comment]
After this PR got merged we can't build flutter-tizen with native(cpp) host.
I'm working on this.
```
  Invoking: C/C++ Linker
"/home/boram/tizen-studio/tools/smart-build-interface/../llvm-10/bin/clang++" -o ./Debug/runner @./Debug/target_objs.file   -Llib
-L"/home/boram/Work/f-project/sample/app_native_1/tizen/./lib" -Wl,--no-undefined  -Wl,-E -target i586-tizen-linux-gnueabi -gcc-toolchain
"/home/boram/tizen-studio/tools/smart-build-interface/../i586-linux-gnueabi-gcc-9.2/" -ccc-gcc-name i586-linux-gnueabi-g++ -march=i586
-Wl,-z,relro,--as-needed,-z,noseparate-code -static-libstdc++ -Xlinker --as-needed -pie -lpthread
--sysroot="/home/boram/tizen-studio/platforms/tizen-4.0/mobile/rootstraps/mobile-4.0-emulator.core" -Xlinker --version-script="./.exportMap"
-L/home/boram/tizen-studio/platforms/tizen-4.0/mobile/rootstraps/mobile-4.0-emulator.core/usr/lib @./Debug/platform_libs.file -Xlinker
-rpath='$ORIGIN/../lib' -Werror-implicit-function-declaration "-Wl,--unresolved-symbols=ignore-in-shared-libs" -lflutter_tizen_mobile
-L"/home/boram/Work/f-project/sample/app_native_1/tizen/lib"
-I"/home/boram/Work/f-project/flutter-tizen/flutter/bin/cache/artifacts/engine/tizen-common/cpp_client_wrapper/include"
-I"/home/boram/Work/f-project/flutter-tizen/flutter/bin/cache/artifacts/engine/tizen-common/public"
-I"/home/boram/Work/f-project/flutter-tizen/embedding/cpp/include" "-Wl,--whole-archive"
"/home/boram/Work/f-project/flutter-tizen/embedding/cpp/Debug/libembedding_cpp.a" "-Wl,--no-whole-archive" -lappcore-agent -lcapi-appfw-app-common
-lcapi-appfw-application -ldlog
-I"/home/boram/Work/f-project/sample/app_native_1/.dart_tool/flutter_build/aa7866749bd28d8c85c7d34228bfd874/tizen_plugins/include"
/home/boram/tizen-studio/tools/smart-build-interface/../i586-linux-gnueabi-gcc-9.2/lib/gcc/i586-tizen-linux-gnueabi/9.2.0/../../../../i586-tizen-l
inux-gnueabi/bin/ld: /home/boram/Work/f-project/flutter-tizen/embedding/cpp/Debug/libembedding_cpp.a(elm_flutter_view.o): in function
`ElmFlutterView::Resize(int, int)':
/home/boram/Work/f-project/flutter-tizen/embedding/cpp/elm_flutter_view.cc:62: undefined reference to `evas_object_geometry_get'
/home/boram/tizen-studio/tools/smart-build-interface/../i586-linux-gnueabi-gcc-9.2/lib/gcc/i586-tizen-linux-gnueabi/9.2.0/../../../../i586-tizen-l
inux-gnueabi/bin/ld: /home/boram/Work/f-project/flutter-tizen/embedding/cpp/Debug/libembedding_cpp.a(elm_flutter_view.o): in function
`ElmFlutterView::GetWidth()':
/home/boram/Work/f-project/flutter-tizen/embedding/cpp/elm_flutter_view.cc:73: undefined reference to `evas_object_geometry_get'
/home/boram/tizen-studio/tools/smart-build-interface/../i586-linux-gnueabi-gcc-9.2/lib/gcc/i586-tizen-linux-gnueabi/9.2.0/../../../../i586-tizen-l
inux-gnueabi/bin/ld: /home/boram/Work/f-project/flutter-tizen/embedding/cpp/Debug/libembedding_cpp.a(elm_flutter_view.o): in function
`ElmFlutterView::GetHeight()':
/home/boram/Work/f-project/flutter-tizen/embedding/cpp/elm_flutter_view.cc:81: undefined reference to `evas_object_geometry_get'
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [/home/boram/tizen-studio/tools/ide/resources/native/Build/makefile.mk:135: Debug/runner] Error 1
make[1]: Leaving directory '/home/boram/Work/f-project/sample/app_native_1/tizen'
make: *** [/home/boram/tizen-studio/tools/ide/resources/native/Build/makefile:27: app_build] Error 2
```

**Pull request merged**